### PR TITLE
Added: nala (and nala-legacy)

### LINKED ISSUE #381: Add nala
## Requesting a new app ✨

 - [x] I have checked that this app does not already have a pending request and has not previously been declined.
 - App Name: Nala
 - Download URL: https://gitlab.com/volian/nala/-/releases
 - Website URL: https://gitlab.com/volian/nala
 - App Summary: Commandline frontend for the APT package manager
 - Notes: Nala have two variants, the `nala` for Ubuntu 22.04 / Debian Sid and newer, and `nala-legacy` for Ubuntu 21.04 / Debian Stable and older.

## Suggested include:
```
function deb_nala() {
    ARCHS_SUPPORTED="amd64 arm64 armhf i386"
    GPG_KEY_URL="https://deb.volian.org/volian/scar.key"
    APT_LIST_NAME="nala"
    APT_REPO_URL="deb [arch=${HOST_ARCH}, signed-by=/usr/share/keyrings/${APT_LIST_NAME}-archive-keyring.gpg] http://deb.volian.org/volian/ scar main"
    PRETTY_NAME="Nala"
    WEBSITE="https://gitlab.com/volian/nala"
    SUMMARY="Commandline frontend for the APT package manager for Ubuntu 22.04 / Debian Sid and newer."
}

function deb_nala-legacy() {
    ARCHS_SUPPORTED="amd64 arm64 armhf i386"
    GPG_KEY_URL="https://deb.volian.org/volian/scar.key"
    APT_LIST_NAME="nala-legacy"
    APT_REPO_URL="deb [arch=${HOST_ARCH}, signed-by=/usr/share/keyrings/${APT_LIST_NAME}-archive-keyring.gpg] http://deb.volian.org/volian/ scar main"
    PRETTY_NAME="Nala Legacy"
    WEBSITE="https://gitlab.com/volian/nala"
    SUMMARY="Commandline frontend for the APT package manager for Ubuntu 21.04 / Debian Stable and older.
}
```

### PR DESCRIPTION
**Pull request by philclifford**
Closes #381 
by implementing the solution given there

### CODE REVIEW

**Comment by natanjunges** [comment]
Please rebase with main and update to the new package metadata format.

**Review by natanjunges — CHANGES_REQUESTED** [review]
Please rebase with main and update to the new package metadata format.

**Comment by philclifford** [comment]
> Please rebase with main and update to the new package metadata format.

I'd love to but main is broken: (`./deb-get prettylist` fails so `cog` on the README fails) 
`prettylist` is not tested in the current test btw - (and neither is cog functionality)

Here's what happened (this is on my "clean" upstream branch as a check after getting the same on my `add-nala-squashed` branch)
````
04:52 $ git pull -r upstream main 
From github.com:wimpysworld/deb-get
 * branch            main       -> FETCH_HEAD
Updating 39d40d8..687a3dc
Fast-forward
 CONTRIBUTING.md |  11 +++--
 README.md       |  12 ++++-
 deb-get         | 498 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++-------------------------------------------------------------------
 debian/control  |   1 +
 debian/postinst |  20 +++++++++
 5 files changed, 372 insertions(+), 170 deletions(-)
 create mode 100755 debian/postinst
(base) ✔ ~/src/deb-get [official L|…7⚑ 11] 
04:52 $ ./deb-get prettylist
| Source   | Package Name   | Description   |
| :------: | :------------- | :------------ |
| [<img src=".github/debian.png" align="top" width="20" />](https://1password.com/) | `1password` | <i>The easiest way to store and use strong passwords.</i> |
| [<img src=".github/github.png" align="top" width="20" />](https://github.com/OrangeDrangon/android-messages-desktop/) | `android-messages-desktop` | <i>Run Android Messages as a desktop app.</i> |
  [!] ERROR! Missing required information of github package antimicrox:
URL=
VERSION_PUBLISHED=
(base) ✘-1 ~/src/deb-get [official L|…7⚑ 11] 
04:52 $ cog -r README.md 
Cogging README.md
Traceback (most recent call last):
  File "README.md", line 152, in <module>
    pretty_list = subprocess.check_output(["./deb-get", "prettylist"], encoding="utf-8")
  File "/home/phil/anaconda3/lib/python3.9/subprocess.py", line 424, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/home/phil/anaconda3/lib/python3.9/subprocess.py", line 528, in run
    raise CalledProcessError(retcode, process.args,
CalledProcessError: Command '['./deb-get', 'prettylist']' returned non-zero exit status 1.
(base) ✘-4 ~/src/deb-get [official L|…7⚑ 11] 

```

**Comment by philclifford** [comment]
when #544 is applied this will be updated and ready.
I'll try and fake it ..

**Review by natanjunges — APPROVED** [review]

File: deb-get [hunk 1]
```diff
@@ -699,6 +699,28 @@ function deb_code() {
     SUMMARY="Code editing. Redefined."
 }
 
+function deb_nala() {
+    ARCHS_SUPPORTED="amd64 arm64 armhf i386"
+    GPG_KEY_URL="https://deb.volian.org/volian/scar.key"
+    APT_LIST_NAME="nala"
+    APT_REPO_URL="deb  http://deb.volian.org/volian/ scar main"
```
Comment (natanjunges) [inline deb-get:706]: The new format does not have the leading `deb`.

**Review by natanjunges — CHANGES_REQUESTED** [review]

File: deb-get (hunk 1 above)
Comment (philclifford) [reply deb-get]: oops sorry: missed that. Sorted

**Review by natanjunges — APPROVED** [review]

**Pull request merged**